xbmc/Application.cpp

```diff
@@ -1117,6 +1117,7 @@ bool CApplication::Initialize()
     // check if we should use the login screen
     if (m_ServiceManager->GetProfileManager().UsingLoginScreen())
     {
+      CServiceBroker::GetPeripherals().Initialise();
```

Comment:
@afedchin is this line needed? when I enable the login screen CPeripherals::Initialise() gets called twice